src/pymatgen/io/vasp/sets.py

```diff
@@ -2679,12 +2679,20 @@ class LobsterSet(VaspInputSet):
     user_potcar_functional: UserPotcarFunctional = "PBE_54"
 
     CONFIG = MPRelaxSet.CONFIG
-    _valid_potcars: Sequence[str] | None = ("PBE_52", "PBE_54")
+    _valid_potcars: Sequence[str] | None = ("PBE_52", "PBE_54", "PBE_64")
 
     def __post_init__(self) -> None:
         super().__post_init__()
         warnings.warn("Make sure that all parameters are okay! This is a brand new implementation.")
 
+        if self.user_potcar_functional == "PBE_64":
```

Comment:
```suggestion
        if self.user_potcar_functional == "PBE_64":
```

Could you add PBE_52 here as well? Thanks!

Reply:
Done. Thank you for reminding me.

Reply:
could you resolve the merge conflicts? `sets.py` is now split into submodules so it's a matter of re-applying your changes to the right file

Reply:
This has been reverted by https://github.com/materialsproject/pymatgen/pull/3989.